docs/findings/el7.md

```diff
@@ -7,199 +7,89 @@
+## Install McAfee Virus Scanning Software
 
-### `/var/tmp` — `nosuid` Option Missing
+Conditionally-Valid Finding: 
 
-Cannot satisfy: `/var/tmp` not implemented as stand-alone filesystem.
+* Where configured to do so, watchmaker will install HBSS or VSEL. Any scan-findings on systems watchmaker has been configured to install HBSS or VSEL are typically due to version mismatches between installed and scanned-for versions
+* Where required/scanned for but not installed, site will need to specify automatable installation-method that will produce match againste scanned-for configuration
+* Where not required, scanner should either be reconfigured not to scan for presence or scan-results should be ignored
 
-Typically, STIG guidance recommends not that `/var/tmp` be a truly standalone filesystem but as a bind-mount of `/var/tmp`. When so implemented, `/var/tmp` inherits the same mount-options used for `/tmp`. Industry best-practices around the implementation of `/tmp` and `/var/tmp` is that the former is ephemeral — contents disappear at each boot and/or by way of a "`/tmp` cleaner" process &dash; while the contents of `/var/tmp` are non-ephemeral. Further, most Enterprise Linux users expect the contents of `/var/tmp` to be non-ephemeral. Because `/tmp` is a VMS-based pseudo-filesystem, the only way to meet both the industry best-practices guidance and user assumptions for `/var/tmp` is to _not_ make `/var/tmp` a bind-mount of `/tmp`. Because the STIGs fail to specify "`/var` must be a standalone filesystem" and not wishing to waste further disk-space to create a `/var/tmp` separate from the `/var` filesystem, `/var/tmp` is typically left as a subdirectory of the `/var` filesystem. This means that `/var/tmp` has no mount options of its own. The only appicable mount options are those inherited from the parent `/var` filesystem.
+## Enable FIPS Mode in GRUB2
 
-Note: for programs that place the additional requirement that `/var/tmp` _must_ be a standalone filesystem, it is recommended to up the size of the root volume-group, create an LVM volume to host the `/var/tmp` filesystem and then apply the site-local mount-options to the filesystem. This can all be accomplished at system provisioning-time, but is outside the scope of the watchmaker utility to effect.
+Conditionally-Valid Finding: Both spel and watchmaker implement `fips=1` by default. If finding occurs, either:
 
-## Missing Audit Rules
+* There is an error in scanner's validation-method
+* System has been intentionally de-configured for FIPS &mdash; typically due to hosted-software's requirements &mdash; and will need to be POA&Med.
```

Comment:
```suggestion
* System has been intentionally de-configured for FIPS &mdash; typically due to hosted-software's requirements &mdash.
```

The POA&M term is very domain-specific. I think we should remove it or generalize.

Reply:
"Short-term deployment security-exception"?